### DISCUSSION TITLE
Nas shares not mounted if NAS not pingable

### DISCUSSION THREAD

**Discussion by vale-max** [rern/rAudio#1666] [Fixed Issues]
The NAS shares are set as noauto in fstab and then mounted by a script at startup, however the script mounts the shares only if the nas responds to ping.
It is possible that a firewall does not allow IGMP but allows the share protocol, in this case the mount would be possible but it fails to be done.
Would be better if the script does not try to ping the NAS before mounting the shares

**Comment by pana800** [comment]
I faced the similar issue. Had to press Remount button after every reboot. removing 'noauto' and even adding '_netdev' in the fstab didn't help.
Tried temporary fix by adding the service file in the systemd/system. It worked.

**Comment by rern** [comment]
@vale-max, @pana800,

Please try `arping SERVER_IP` whether it pass the firewall of the server.

**Comment by vale-max** [comment]
Timeout for me

**Comment by vale-max** [comment]
@rern for cifs you could use the return code of
nc -z nasaddress.xxx 445
